src/services/game.ts

```diff
@@ -4,12 +4,13 @@ import { Food } from '../models/food';
 import { AntLoaderService } from './ant-loader';
 import { AntAction } from '../models/ant-action';
 import { Game } from '../models/game';
+import { GameStatus } from '../models/game-status';
 
 const GRID_WIDTH = +(process.env.GRID_WIDTH || 200);
 const GRID_HEIGHT = +(process.env.GRID_HEIGHT || 80);
 const FOOD_PERCENTAGE = +(process.env.FOOD_PERCENTAGE || 0.05);
 export const MAX_TICKS = +(process.env.MAX_TICKS || 1000);
-export const TICKS_PER_SECOND = +(process.env.TICKS_PER_SECOND || 2);
+export const TICKS_PER_SECOND = +(process.env.TICKS_PER_SECOND || 100);
```

Comment:
I feel like 100 might be pretty high for a default?

Reply:
yeah, I was running tournaments and forgot to put this back to a reasonable number

is 5 good?

Reply:
Yeah that seems fine

Reply:
> > i think we could get rid of the config endpoint at this point? yeah?
> 
> I thought the UI used it for some stuff still? Maybe this new endpoint has everything it needs, I don't remember....

yeah, i want games to be able to be running with different board sizes / lengths.

I changed the tournament grinder to use the gamestatus endpoint, but the UI still needs it for now.

Reply:
> found some bugs, will need to do some more refactoring before this is sorted, i was trying to do this in stages, but it would most likely be better to handle / test it all at once

ok, i think i have the bugs worked out, verified that it all works using docker